client/src/i18n/fr/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "FUNCTION" : {
```

Comment:
Since this is no longer translated as "Function", we should make all the keys "PROFESSION" or something similar.  This can be a separate PR if you wish.